src/components/project-details/project-details.test.tsx

```diff
@@ -4,18 +4,29 @@
  */
 
 import { render, screen } from '@testing-library/react';
-import { Project } from 'utils';
+import { exampleProject1, exampleProject2, exampleProject3 } from 'utils';
 import ProjectDetails from './project-details';
 
-it('Renders title', () => {
-  const exProject: Project = {
-    wbsNum: '1.1.0',
-    name: 'Impact Attenuator',
-    projectLead: 'Person Allen',
-    projectManager: 'Person Gilbert',
-    duration: 2
-  };
-  render(<ProjectDetails project={exProject} />);
-  const titleElement = screen.getByText(/Project Details/i);
-  expect(titleElement).toBeInTheDocument();
+describe('project details component', () => {
+  it('Renders title', () => {
+    render(<ProjectDetails project={exampleProject1} />);
+    const titleElement = screen.getByText('Project Details');
+    expect(titleElement).toBeInTheDocument();
+  });
+
+  it('Renders WBS#', () => {
+    render(<ProjectDetails project={exampleProject2} />);
+    const projectElement = screen.getByText(
+      `${exampleProject2.wbsNum.area}.${exampleProject2.wbsNum.project}.${exampleProject2.wbsNum.workPackage}`
+    );
```

Comment:
```suggestion
    const projectElement = screen.getByText(wbsPipe(exampleProject2.wbsNum));
```
use the wbsPipe for this instead, and make sure to import the wbsPipe at the top too.